src/Data/Nat/Bounded/Properties.agda

```diff
@@ -23,15 +24,16 @@ open import Relation.Binary.Structures using (IsPartialEquivalence; IsEquivalenc
 open import Relation.Binary.Core using (_⇒_)
 open import Relation.Binary.Definitions
 open import Relation.Binary.PropositionalEquality
-  using (_≡_; refl; erefl; sym; trans; cong; subst; _≗_)
+  hiding (isEquivalence; setoid)
```

Comment:
going from 'using' to 'hiding', you've now pulled in a lot more of the namespace and it might come back to haunt you later.

Reply:
Yes, you're right. Reverted, with the addition of `module ≡-Reasoning`.